server_src/lib/serverCommands.js

```diff
@@ -19,23 +23,39 @@ class SystemCommands {
 
     // If we're on pm2, then restart buddy!
     if (isPm2()) {
-      try{
-        await exec("pm2 restart OctoFarm")
-        checkForNamedService = true;
-      }catch (e){
+      try {
+        let doesFunctionExist = await checkIfFunctionExistsInPath("pm2");
+
+        if (doesFunctionExist) {
+          setTimeout(async () => {
```

Comment:
Shouldnt the timeout be awaited as well?

Reply:
Wrong commit anyway